README.md

```diff
@@ -1,14 +1,15 @@
 # Acquia Drupal Recommended Settings
-The Acquia Drupal Recommended Settings plugin adds the recommended settings to the Drupal project, so developers won't have to edit settings.php manually.
+The Acquia Drupal Recommended Settings plugin adds the recommended settings to
+the Drupal project, so developers won't have to edit settings.php manually.
 
 The recommended settings includes:
-- the required database credentials
+- the required database credentials.
```

Comment:
If these are pointers then first word of each should be in caps.
- The required database credentials.
- Configuration sync directory path.
-  File directory path i.e public/private etc.
- Acquia site studio sync directory path.

Reply:
done